packages/@webex/plugin-meetings/src/meetings/index.ts

```diff
@@ -335,8 +336,13 @@ export default class Meetings extends WebexPlugin {
           }
         });
     } else {
-      meeting.locusInfo.parse(meeting, data);
+      return new Promise((resolve) => {
+        meeting.locusInfo.parse(meeting, data);
+        resolve(undefined);
+      });
     }
+
+    return Promise.resolve(undefined);
```

Comment:
as you've added the `async` keyword to the function, then it automatically always returns a promise resolved with whatever return value was returned, so you don't need to convert all the return statements in the function to return a promise - the function `handleLocusEvent()` will always return a promise anyway

Reply:
Thanks @marcin-bazyl 

The eslint actually threw [consistent-return](https://eslint.org/docs/latest/rules/consistent-return) error. That's why I made this change. Turns out, it accepts `return undefined`

Made this change.